patroni/ctl.py

```diff
@@ -628,7 +628,7 @@ def failover(config_file, cluster_name, master, candidate, force, dcs, scheduled
         logging.warning('Failing over to DCS')
         click.echo(timestamp() + ' Could not failover using Patroni api, falling back to DCS')
         click.echo(timestamp() + ' Initializing failover from master {0}'.format(master))
-        dcs.manual_failover(master, candidate, scheduled_at=failover_value)
```

Comment:
You are right, there is a bug on the line 631, it should look like:
```python
dcs.manual_failover(master, candidate, scheduled_at=scheduled_at)
```

But change on the line 457 is not needed and even dangerous.